.github/workflows/release.yml

```diff
@@ -1,46 +1,49 @@
-name: Create, publish container image
+name: Docker
```

Comment:
@akshay196 what's do we want to achieve here ?

Reply:
If you're asking about name change then, there is nothing specific achieved by changing workflow name apart of how it is displayed in Checks output:

![Screenshot from 2023-04-13 20-34-37](https://user-images.githubusercontent.com/16691314/231802776-60b75185-3160-457c-9093-ae4ad3bdd5d9.png)